### DISCUSSION TITLE
Python 3 scripts to backup a minecraft server

### DISCUSSION THREAD

**Discussion by slyfox1186** [itzg/docker-minecraft-server#2589] [Ideas]
I wrote two Python 3 scripts to back up a Minecraft server on docker. You use the container ID for both scripts to identify the correct docker image. If anyone would like some help customizing these let me know.

[backup-generic](https://github.com/slyfox1186/script-repo/blob/main/Python3/backup-docker-minecraft-server-generic.py)

[Backup-with-Twilio](https://github.com/slyfox1186/script-repo/blob/main/Python3/backup-docker-minecraft-server-with-twilio.py)

Now create a crontab file that points to either of these scripts and let it run.

Do you guys have any cools scripts that you use?

**Comment by slyfox1186** [comment]
@itzg if you want to utilize this you are more than welcome. I love this docker repo!

**Reply by itzg** [reply]
I appreciate the offer but I already have this that I maintain 

https://github.com/itzg/docker-mc-backup

**Comment by slyfox1186** [comment]
Oh awesome! I’ll check it out.

**Comment by slyfox1186** [comment]
nice! Ill check this out!
